ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1069,6 +1069,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-p3-f28-long-duration-workload
```

Comment:
@kuiwang02 Shall we merge this into `aws-ipi-p3-f28-long-duration-part3-3` ?

Reply:
@liangxia I do not suggest it because workload (oc-mirror) cases take so long duration too in the future (more and more oc-mirror case will be automated).
current [aws-ipi-p3-f28-long-duration-part3-3](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-p3-f28-long-duration-part3-3/1635485263532658688) is up to 14h.

if combining them, it is possible to more than 20h (even 22h, 24h).

So, I do not suggest it.

Reply:
Ack